pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/config/index.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: layout.pug
+navigationTitle: Managing Alertmanager Configurations
+title: Managing Alertmanager Configurations
+menuWeight: 20
+excerpt: Enabling and configuring Alertmanager
+render: mustache
+model: ../../../data.yml
+---
+
+# Enabling Alertmanager
+
+Alertmanager is off by default.
+To enable Alertmanager, the {{ model.techName }} service must be configured to load the Alertmanager configurations from a Git repository.
+See the [Alertmanager documentation](https://prometheus.io/docs/alerting/configuration/) to learn how to create the configuration file.
+
+## Save Alertmanager configurations
+
+The Alertmanager configuration file (YAML format) must be named `config.yml`.
+You should save `config.yml` and all template files in a Git repository.
+Assume that the repository is `https://github.com/company/alertmanager-configs`.
+
+You can put `config.yml` and template files in a folder in the repository.
+For instance, `https://github.com/company/alertmanager-configs/production`.
+
+```json
+{
+  "alertmanager": {
+    "config_repository": {
+      "url": "https://github.com/company/alertmanager-configs",
+      "path": "/production"
+    }
+  }
+}
+```
+
+The Git repository can be either public or private.
+You can omit the `credentials` section if the Git repository is public.
+If the Git repository is private, you must configure the credentials to access the Git repository.
+
+## Create secrets for Git repository credentials
+
+If the Git repository containing the Alertmanager configurations is private, you must configure the secrets first.
+Currently, the following Auth types are supported.
+
+<p class="message--important"><strong>IMPORTANT: </strong>If the Git repository contains submodules, all the submodules must use the same Auth type as what's used for the repository. For example, if HTTP Auth is configured for the Git repository, all of the submodules in the repository's <code>.gitmodules</code> file must use the <code>https</code> URL scheme.</p>
+
+### HTTP Auth
+
+```bash
+dcos security secrets create --value=<GITHUB_USERNAME> githubusername-secret
+dcos security secrets create --value=<GITHUB_PASSWORD> githubpassword-secret
+```
+
+For GitHub, the password can be the API token.
+
+Create a custom option file (`options.json`) like the following.
+
+```json
+{
+  "alertmanager": {
+    "config_repository": {
+      "url": "https://github.com/company/alertmanager-configs",
+      "path": "/production",
+      "credentials": {
+        "username_secret": "githubusername-secret",
+        "password_secret": "githubpassword-secret"
+      }
+    }
+  }
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's, the public key) to the repository.
```

Comment:
The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)